server/controllers/defaultBoardContents/defaultBoardContents.json

```diff
@@ -0,0 +1,30 @@
+{
+  "boards": {
+    "boardTitle": "New Board",
+    "boardId": "d93bbef3-ca27-4767-9d3d-acf17e29cc17",
```

Comment:
So, I think leaving boardId blank is totally fine, Mongo will generate ids for us.